apsuite/commisslib/meas_bo_tune_by_bpm.py

```diff
@@ -82,13 +82,13 @@ def configure_bpms(self):
         trigbpm.source = prms.trigger_source
         trigbpm.nr_pulses = prms.nr_pulses
 
-    def get_orbit_data(self, injection=False, external_trigger=False):
-        """Get orbit data from BPMs in TbT acquisition rate
-        BPMs must be configured to listen DigBO event and the DigBO
-        event must be in Injection mode.
+    def get_orbit(self, injection=False, external_trigger=False):
+        """Get orbit data from BPMs in TbT acquisition rate..
         If injection is True, then injection is turned on before the measure.
         If external_trigger is True, the event will listen a external trigger.
         """
+        if not hasattr(self, 'data'):
```

Comment:
why is this needed? For this type of object the `data` dictionary is always defined at initialization.

Reply:
Thanks for the observation, I didn't figure out that. 
I removed these lines.